website/blog/2018-05-23-1.13.0.md

```diff
@@ -0,0 +1,1119 @@
+---
+author: Lucas Duailibe (@duailibe)
+authorURL: https://twitter.com/duailibe
+title: "Prettier 1.13: Conquering the web!"
+---
+
+This releases adds a new support for several new features and first-class support for working on the browser.
+
+<!--truncate-->
+
+## Highlights
+
+### API/CLI
+
+#### Prettier works in the browser!
+
+This has been long wanted by the team and our users and we're finally announcing Prettier 1.13 has first-class support for running in the browser. In the past it required several hacks, but as of today you can just load the core bundle and the parsers you want to use.
+
+<!-- prettier-ignore -->
+```html
+<script src="https://unpkg.me/prettier@1.13.0/standalone.js"></script>
+<script src="https://unpkg.me/prettier@1.13.0/parser-babylon.js"></script>
+<script src="https://unpkg.me/prettier@1.13.0/parser-graphql.js"></script>
+<script type="text/javascript">
+$("#format").on("click", function() {
+  var input = $("#input").val();
+  $("#input").val(
+    prettier.format(input, { parser: "babylon", plugins: prettierPlugins })
+  );
+});
```

Comment:
Like @azz mentioned in another comment, should we even use jQuery in this example?

```js
var formatButton = document.querySelector("#format");
var textInput = document.querySelector("#input");
formatButton.addEventListener("click", function() {
  var value = textInput.value;
  textInput.value = prettier.format(value, {
    parser: "babylon",
    plugins: prettierPlugins
  });
});
```

Btw, where does `prettierPlugins` come from?

Reply:
It's a global variable.

https://github.com/prettier/prettier/blob/a1e8d586ad4f476cda9f773358caa509a3655701/scripts/build/build.js#L141

Reply:
Is this documented anyhere?

Reply:
In #4550, but perhaps not explicitly.